src/main/scala/cognite/spark/v1/PushdownUtilities.scala

```diff
@@ -379,6 +379,10 @@ object PushdownUtilities {
   }
 }
 
+trait NamedRelation {
```

Comment:
nit: looks unrelated to pushdown. Also, maybe make CdfRelation extend NamedRelation, they can use the same shortname and it makes it more likely that named part is applied to new source types

Reply:
This trait is for Objects so it doesn't really work, AFAIK you can't force classes to have companion objects with a variable

Reply:
right, makes sense

Reply:
Ok, I did the latter, now both the companion object and the relation itself extend namedRelation, I'm not sure if I like it better or not but it can't hurt